backend/test/v2/resources/env-var.yaml

```diff
@@ -46,7 +46,7 @@ deploymentSpec:
           \ *\n\ndef comp(env_var: str) -> str:\n    import os\n\n    value = os.getenv(env_var,\
           \ \"\")\n\n    if value == \"\":\n        raise Exception(\"Env var is not\
           \ set\")\n\n    return value\n\n"
-        image: public.ecr.aws/docker/library/python:3.12
+        image: python:3.12-alpine
```

Comment:
No https://github.com/kubeflow/manifests/blob/1c74b37c023f93996201f8e703d89f47b9ab336a/.github/workflows/pipeline_swfs_test.yaml#L30 is the right way!